### PR TITLE
Maintenance | Removing console warnings trigger from discussion code

### PR DESCRIPTION
**Pull request by bkoval**
Seems like this code was triggering tslint's "no-object-literal" rule to be broken.

### CODE REVIEW

File: server/facets/showApplication.ts [hunk 1]
```diff
@@ -56,13 +56,19 @@ function outputResponse (request: Hapi.Request, reply: Hapi.Response, context: a
 	reply.view('application', context);
 }
 
+/**
+ * @param {string} hostName
+ * @returns {*}
```
Comment (bognix) [inline server/facets/showApplication.ts]: not * but Object

Comment (bognix) [inline server/facets/showApplication.ts]: how about typedef for returned object? like in front/scripts/main/models/CuratedContentEditorModel.js:3

Comment (bkoval) [inline server/facets/showApplication.ts]: Good idea!

File: server/facets/showApplication.ts [hunk 2]
```diff
@@ -56,13 +63,19 @@ function outputResponse (request: Hapi.Request, reply: Hapi.Response, context: a
...
+/**
+ * @param {string} hostName
+ * @returns {CommunityAppConfig}
+ */
 function getDistilledDiscussionsSplashPageConfig(hostName: string): Object {
-	var distilledConfig = {};
-	if (discussionsSplashPageConfig[hostName]) {
-		distilledConfig['androidAppLink'] = discussionsSplashPageConfig[hostName].androidAppLink;
-		distilledConfig['iosAppLink'] = discussionsSplashPageConfig[hostName].iosAppLink;
+	var mainConfig = discussionsSplashPageConfig[hostName];
```
Comment (hakubo) [inline server/facets/showApplication.ts]:
```if (discussionsSplashPageConfig[hostName]) {
var {androidAppLink, iosAppLink} = discussionsSplashPageConfig[hostName];
return {
  androidAppLink,
  iosAppLink
  }
}
```

**Comment by hakubo** [comment]
LGTM

File: server/facets/showApplication.ts (hunk 2 above)
Comment (bkoval) [inline server/facets/showApplication.ts]: This is causing tslint to shout a bit. I'll stick to the current form and we can refactor it when we ES6-ify the code here :)

**Pull request merged**